### PR TITLE
add cu126 torchvision and torchaudio aarch64 build

### PR DESCRIPTION
**Pull request by tinglvv**
related to https://github.com/pytorch/pytorch/issues/138440

cc @atalman @ptrblck @nWEIdia

### CODE REVIEW

**Comment by tinglvv** [comment]
the format of the json files seems off after running tools/tests/update_test_assets.sh.

**Review by atalman — APPROVED** [review]
lgtm

File: tools/tests/assets/build_matrix_macos_wheel.json [hunk 1]
```diff
@@ -1,79 +1 @@
...
...2xlarge", "installation": "pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu", "channel": "nightly", "upload_to_base_bucket": "yes", "stable_version": "2.5.1", "use_split_build": false}, {"python_version": "3.12", "gpu_arch_type": "cpu", "gpu_arch_version": "", "desired_cuda": "cpu", "container_image": "pytorch/manylinux-builder:cpu", "package_type": "wheel", "build_name": "wheel-py3_12-cpu", "validation_runner": "linux.2xlarge", "installation": "pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu", "channel": "nightly", "upload_to_base_bucket": "yes", "stable_version": "2.5.1", "use_split_build": false}, {"python_version": "3.13", "gpu_arch_type": "cpu", "gpu_arch_version": "", "desired_cuda": "cpu", "container_image": "pytorch/manylinux-builder:cpu", "package_type": "wheel", "build_name": "wheel-py3_13-cpu", "validation_runner": "linux.2xlarge", "installation": "pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu", "channel": "nightly", "upload_to_base_bucket": "yes", "stable_version": "2.5.1", "use_split_build": false}]}
```
Comment (kit1980) [inline tools/tests/assets/build_matrix_macos_wheel.json:1]: Can we have all the json files formatted as before?

**Comment by atalman** [comment]
@tinglvv to have proper formatting please run: 
```
python -m tools.tests.test_generate_binary_build_matrix --update-reference-files
```

File: tools/tests/assets/build_matrix_macos_wheel.json (hunk 1 above)
Comment (tinglvv) [reply tools/tests/assets/build_matrix_macos_wheel.json]: thanks, yes, done

**Pull request merged**